dev-docs/bidders/taboola.md

```diff
@@ -27,14 +28,14 @@ The Taboola Adapter requires setup before beginning. Please contact us at prebid
 
 {: .table .table-bordered .table-striped }
 
-| Name          | Scope    | Description                                             | Example                    | Type         |
-|---------------|----------|---------------------------------------------------------|----------------------------|--------------|
-| `tagId`       | required | Tag ID / Placement Name <br>                            | `'Below The Article'`      | `String`     |
-| `publisherId` | required | Numeric Publisher ID <br>(as provided by Taboola)       | `'1234567'`                | `String`     |
-| `bcat`        | optional | List of blocked advertiser categories (IAB)             | `['IAB1-1']`               | `Array`      |
-| `badv`        | optional | Blocked Advertiser Domains                              | `'example.com'`            | `String Url` |
-| `bidfloor`    | optional | CPM bid floor                                           | `0.25`                     | `Float`      |
-| `endpointUrl` | optional | Endpoint Url (only if provided by Taboola)              | `https://example.com`      | `String`     |
+| Name              | Scope    | Description                                       | Example               | Type         |
+|-------------------|----------|---------------------------------------------------|-----------------------|--------------|
+| `tagId`           | required | Tag ID / Placement Name <br>                      | `'Below The Article'` | `String`     |
+| `publisherId`     | required | Numeric Publisher ID <br>(as provided by Taboola) | `'1234567'`           | `String`     |
+| `publisherDomain` | optional | Publisher Domain                                  | `'example.com'`       | `String`     |
```

Comment:
does your client side adapter support this new publisherDomain parameter? If not, you need to add a comment in the description like "(server-side adapter only)"

Reply:
@bretg 
added a comment. 
thank you.